json-server/routes.json

```diff
@@ -1,4 +1,5 @@
 {
     "/api/v1/*": "/$1",
-    "/firms/:id": "/firms"
+    "/firms/:id": "/firms",
+    "/firms/:firmId/editPii": "/edit"
```

Comment:
There isn't a route with "editPi" called in the code and there isn't an "edit" object in the db either